.gitignore

```diff
@@ -0,0 +1,22 @@
+.idea/
+env/
+out/
+data
+
+__pycache__/
+*.trk
+*.gz
+*.pt
+*.png
+*.comet.config
+*.npy
+
+# Unit test / coverage reports
+.coverage
+
+log
+*/.ipynb_checkpoints
+
+setenv.sh
```

Comment:
We do not have such a script in our code, right?

Reply:
It should be removed now

Reply:
Done